api/test/integration/conftest.py

```diff
@@ -403,27 +389,29 @@ def clean_up_env():
         enable_white_mode()
 
     general_procedure(module)
-    values = build_and_up(interval=10, build=request.config.getoption('--nobuild'), env_mode=env_mode)
+    build_and_up(build=request.config.getoption('--nobuild'), env_mode=env_mode)
```

Comment:
This was right in the end, the variable type is `store_false` meaning that it's always True unless specified.